src/components/StudyList/Name.js

```diff
@@ -26,12 +26,30 @@ const StudyName = ({study}) => {
           textAlign="left"
           data-cy="study name"
         >
+          <Rating
```

Comment:
This looks ok though we may decide to use the `star` and `star outline` icons in the future as an alternative.